x-pack/test/functional/services/aiops/explain_log_rate_spikes.ts

```diff
@@ -56,9 +58,93 @@ export function ExplainLogRateSpikesProvider({ getService }: FtrProviderContext)
       await testSubjects.existOrFail(`aiopsNoWindowParametersEmptyPrompt`);
     },
 
+    async assertNoResultsFoundEmptyPromptExist() {
+      await testSubjects.existOrFail(`aiopsNoResultsFoundEmptyPrompt`);
+    },
+
+    async clickDocumentCountChart() {
+      await elasticChart.waitForRenderComplete();
+      const el = await elasticChart.getCanvas();
+
+      await browser.getActions().move({ x: 0, y: 0, origin: el._webElement }).click().perform();
+    },
+
+    async clickRerunAnalysisButton(shouldRerun: boolean) {
+      await testSubjects.clickWhenNotDisabled(
+        `aiopsRerunAnalysisButton${shouldRerun ? ' shouldRerun' : ''}`
+      );
+
+      await retry.tryForTime(30 * 1000, async () => {
+        await testSubjects.existOrFail(
+          `aiopsRerunAnalysisButton${!shouldRerun ? ' shouldRerun' : ''}`
+        );
+      });
+    },
+
+    async assertAnalysisSectionExist() {
+      await retry.tryForTime(5000, async () => {
+        await testSubjects.existOrFail(`aiopsExplainLogRateSpikesAnalysis`);
+      });
+    },
+
+    async assertRerunAnalysisButton(shouldRerun: boolean) {
```

Comment:
We might want to call this method `assertRerunAnalysisButtonExists` to clarify the type of assertion.

Reply:
Fixed in [e0ca68c](https://github.com/elastic/kibana/pull/138661/commits/e0ca68cdcecca9fe28b39d464a2600e191eac5ad).